soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk64f12

```diff
@@ -8,6 +8,34 @@ if SOC_MK64F12
 config SOC
 	default "mk64f12"
 
+config K6X_CORE_CLOCK_DIVIDER
```

Comment:
I'm fine with the renaming, but please move these configs back to `soc/arm/nxp_kinetis/k6x/Kconfig.soc`. New symbols shouldn't be defined in defconfig files

Reply:
@MaureenHelm Thanks, rearranged it now. As the defaults are maintained for K64, for `CONFIG_K6X_FLEXBUS_CLOCK_DIVIDER=2` this values is moved to board defconfig.